scripts/repo-util

```diff
@@ -34,7 +34,7 @@ def add_manifest(projects: dict):
         git_cmd = f"git -C {path} config --get remote.origin.url"
         url = subprocess.check_output(git_cmd.split(' '), text=True).rstrip()
         repo = removesuffix(url.split('/')[-1], '.git')
-        projects[repo] = path
+        projects[repo.lower()] = (path, repo)
```

Comment:
maybe add a comment in the code that lower case name is the preferred sanitized form?

Reply:
yes, good idea. So far the only info on all this is in the commit message.